server/container/mailbox-jmx/src/main/java/org/apache/james/adapter/mailbox/ReIndexerManagement.java

```diff
@@ -51,8 +52,8 @@ public void reIndex(String namespace, String user, String name) throws MailboxEx
                      .addContext(MDCBuilder.ACTION, "reIndex")
                      .build()) {
             TaskId taskId = taskManager.submit(reIndexer.reIndex(new MailboxPath(namespace, user, name)));
-            taskManager.await(taskId);
-        } catch (IOException e) {
+            taskManager.await(taskId, Duration.ofMillis(Long.MAX_VALUE));
```

Comment:
Do we "really" want to be waiting indefinitly like this? 

Can't we make that a parameter or find a more suited value (1 year?)